core/server/api_container/server/api_container_connect_service.go

```diff
@@ -0,0 +1,95 @@
+package server
```

Comment:
will add implementation in next; combined with ui integration so we should have everything very soon!